Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CovidCertificateConfigMapper.kt

```diff
@@ -51,7 +53,7 @@ class CovidCertificateConfigMapper @Inject constructor() : CovidCertificateConfi
 
     data class CovidCertificateConfigContainer(
         override val testCertificate: CovidCertificateConfig.TestCertificate = TestCertificateConfigContainer(),
-        override val expirationThresholdInDays: Int = 1
```

Comment:
I put that in here deliberately, not sure what the fallback should be.

Reply:
I changed it to 14 days based on:
https://github.com/corona-warn-app/cwa-app-android/pull/3799#discussion_r676476499
and added some unit tests.